File: quark/test/emit/strings/js/strings.js
var _qrt = require("datawire-quark-core");

function main() {
    _qrt.print("\"");
    _qrt.print("\"\"");
    _qrt.print("\u0000");
    _qrt.print("\u00FF");
    _qrt.print("\u00ff");
    _qrt.print("\u00FF");
    _qrt.print("\u00ff");
    _qrt.print("\n");
}
exports.main = main;

main();

File: quarkc/test/emit/expected/rb/strings/lib/strings.rb
module Quark
require "quark"
def self.strings; Strings; end
module Strings
require_relative 'strings' # 0 () ()


def self.call_main()
    self.main(::DatawireQuarkCore::List.new(ARGV))

    nil
end
def self.main(args)
    
    ::DatawireQuarkCore.print("\"")
    ::DatawireQuarkCore.print("\"\"")
    ::DatawireQuarkCore.print("\u0000")
    ::DatawireQuarkCore.print("\u00FF")
    ::DatawireQuarkCore.print("\u00ff")
    ::DatawireQuarkCore.print("\u00FF")
    ::DatawireQuarkCore.print("\u00ff")
    ::DatawireQuarkCore.print("\n")


    nil
end

if __FILE__ == $0 then ::Quark.strings.call_main() end

end # module Strings
end # module Quark
